Merchant/App/Settings/SettingsViewController.swift

```diff
@@ -14,18 +14,18 @@ class SettingsViewController: BDCViewController {
     
     fileprivate let cellId = "settingsCell"
     
-    var tableView: UITableView = {
+    let tableView: UITableView = {
```

Comment:
fileprivate please

Reply:
In this commit was the objective to correct the value type only. Talking about the access control I would leave this for another commit which will do only that as there are possibly many more places than just these...